Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -2921,6 +2949,9 @@ namespace AzToolsFramework
                 }
             );
 
+            // This action is only accessible outside of Component Modes
+            m_actionManagerInterface->AssignModeToAction(DefaultActionContextModeIdentifier, actionIdentifier);
```

Comment:
update: Again I'm wondering about this just being the default and we instead opt in to actions we do want in Component Mode instead - it feels like there would be less chance of error that way around

Reply:
+1 yeah, maybe another convo to follow-up with UX about. In the short-term I'd be up for keeping things consistent with how they are now at the very least, and we can then relax what is available as a separate change so we're not changing multiple things at once

Reply:
Yeah I'm usually just trying to keep things as they are apart from minor things, and record what looks weird for future conversation.